### PR TITLE
Port SmartDashboard - LiveWindow from Screensteps

### LINKED ISSUE #119: SmartDashboard - LiveWindow needs a port
https://wpilib.screenstepslive.com/s/currentCS/m/smartdashboard/c/92707 needs a port

### PR DESCRIPTION
**Pull request by yellowjaguar5**
This is my first pull request on this project, so please let me know if I messed up any of the style standards. Closes #119.

### CODE REVIEW

File: source/docs/software/wpilib-tools/smartdashboard/test-mode-and-live-window/PID-tuning-with-SmartDashboard.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+The F (feedforward) term is used for controlling velocity with a PID controller.
+
+You can find more information in Operating the robot with feedback from sensors(C++ or Java).
+
+Practice Tuning in FRCSim
+-------------------------
+
+With the FRCSim, you can experiment with PID and practice tuning in simulation! This means no broken couplers or time consuming uploads!
+
+A video on tuning PID with SmartDashboard in simulation can be found on the official WPILib channel here: https://www.youtube.com/watch?v=yqD9iHiR3j8
```
Comment (Daltz333) [inline source/docs/software/wpilib-tools/smartdashboard/test-mode-and-live-window/PID-tuning-with-SmartDashboard.rst]:
```suggestion
A video on tuning PID with SmartDashboard in simulation can be found on the official WPILib channel `here <https://www.youtube.com/watch?v=yqD9iHiR3j8>`__
```

File: source/docs/software/wpilib-tools/smartdashboard/test-mode-and-live-window/enabling-test-mode.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Enabling Test mode (LiveWindow)
+===============================
+
+You may add code to your program to display values for your sensors and actuators while the robot is in Test mode. This can be selected from the Driver Station whenever the robot is not on the field. The code to display these values is automatically generated by RobotBuilder and is described in the next article. Test mode is designed to verify the correct operation of the sensors and actuators on a robot. In addition it can be used for obtaining setpoints from sensors such as potentiometers and for tuning PID loops in your code.
+
```
Comment (Daltz333) [inline source/docs/software/wpilib-tools/smartdashboard/test-mode-and-live-window/enabling-test-mode.rst]: Extra line here

Comment (yellowjaguar5) [reply source/docs/software/wpilib-tools/smartdashboard/test-mode-and-live-window/enabling-test-mode.rst]: Removed extra line

**Comment by yellowjaguar5** [comment]
Okay, I checked for extra lines and made all file names lower case. Anything else?

File: source/docs/software/wpilib-tools/smartdashboard/test-mode-and-live-window/pid-tuning-with-smartdashboard.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+------------------------
+.. image:: images/PID-tuning-with-SmartDashboard/tuning-pid-controller.png
+
+Tuning the PID controller can be difficult and there are many articles that describe techniques that can be used. It is best to start with the P value first. To try different values fill in a low number for P, enter a setpoint determined earlier in this document, and note how fast the mechanism responds. If it responds too slowly, perhaps never reaching the setpoint, increase P. If it responds too quickly, perhaps oscillating, reduce the P value. Repeat this process until you get a response that is as fast as possible without oscillation. It's possible that having a P term is all that's needed to achieve adequate control of your mechanism.
+
+Once you have determined P, I, and D values they can be inserted into the program. You'll find them either in the properties for the PIDSubsystem in RobotBuilder or in the constructor for the PID Subsystem in your code.
+
+The F (feedforward) term is used for controlling velocity with a PID controller.
+
+You can find more information in Operating the robot with feedback from sensors(C++ or Java).
```
Comment (Daltz333) [inline source/docs/software/wpilib-tools/smartdashboard/test-mode-and-live-window/pid-tuning-with-smartdashboard.rst]:
```suggestion
.. todo:: add link to pid article when ported

You can find more information in Operating the robot with feedback from sensors.
```

Comment (Daltz333) [reply source/docs/software/wpilib-tools/smartdashboard/test-mode-and-live-window/pid-tuning-with-smartdashboard.rst]: With this change, it's all good. The PID article has yet to be ported (likely to be rewritten).

**Comment by yellowjaguar5** [comment]
@Daltz333 I applied your changes. Anything else?

**Pull request merged**